app-misc/hodie/hodie-1.5.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="a Latin date(1)"
+HOMEPAGE="http://hodie.sourceforge.net"
+SRC_URI="https://github.com/michiexile/${PN}/archive/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+S="$WORKDIR"/${PN}-${P}
+
+src_prepare() {
+	eautoreconf
```

Comment:
Call `default` before `eautoreconf`, because `default` can potentially end up changing files that `eautoreconf` creates.